cost-optimization-aks-content.md

```diff
@@ -0,0 +1,795 @@
+In this document, we'll discuss features and options, and the best practices for running and managing Kubernetes workloads on AKS in a cost optimized way to take advantage of the scalability provided by Azure. 
+
+> [!NOTE]
+> This document assumes you are familiar with Kubernetes, Azure, AKS and autoscaling. If you need a refresher, see the [**Related articles**](#related-articles) section for resources.
+
+## Introduction
+
+As Kubernetes gains widespread adoption, a growing number of enterprises and platform-as-a-service (PaaS) and software-as-a-service (SaaS) providers are using multi-tenant Kubernetes clusters for their workloads. This means that a single cluster might be running applications that belong to different teams, departments, customers, or environments. The multi-tenancy provided by Kubernetes lets companies manage a few large clusters, instead of multiple smaller ones, with benefits such as appropriate resource utilization, simplified management control, and reduced fragmentation.
```

Comment:
Couple small suggestions.  Kubernetes really doesn't have multi-tenancy "provided" -- you kinda need to stitch that together.

```suggestion
As Kubernetes gains widespread adoption, a growing number of enterprises and platform-as-a-service (PaaS) and software-as-a-service (SaaS) providers are using multi-tenant Kubernetes clusters for their workloads. This means that a single cluster is running applications that belong to different teams, departments, customers, or environments. This lets companies manage a few large clusters, instead of multiple smaller ones, with benefits such as appropriate resource utilization, simplified management control, and reduced fragmentation.
```